Makefile

```diff
@@ -99,14 +99,14 @@ $(TARGET_DIR)/%_csr.wasm:	$(CSR_SOURCE_FILES)
 	@touch $@ # Cargo must have a cache somewhere because it doesn't update the file time
 
 $(MERE_MEMORY_WASM):
-	curl --fail -L "https://github.com/mjbrisebois/hc-zome-mere-memory/releases/download/v$(MERE_MEMORY_VERSION)/mere_memory.wasm" --output $@
+	curl --fail -L "https://github.com/holochain/hc-zome-mere-memory/releases/download/v$(MERE_MEMORY_VERSION)/mere_memory.wasm" --output $@
```

Comment:
This URL doesn't open for me. Does it exist but it's private and I don't have access?

Reply:
Seems like the `hc` prefix was stripped during the migration, fixed it